parsec/parsec.c

```diff
@@ -521,14 +521,17 @@ parsec_context_t* parsec_init( int nb_cores, int* pargc, char** pargv[] )
         false, false, vpmap_parameter, &vpmap_parameter);
 
     /* thread binding */
-    parsec_mca_param_reg_int_name("bind", "main_thread", "Force the binding of the thread calling parsec_init",
+    idx = parsec_mca_param_reg_int_name("bind", "main_thread", "Force the binding of the thread calling parsec_init",
                                  false, false, parsec_runtime_bind_main_thread, &parsec_runtime_bind_main_thread);
-    parsec_mca_param_reg_int_name("bind", "threads", "Bind main and worker threads", false, false,
+    parsec_mca_param_reg_syn_name(idx, "runtime", "bind_main_thread", true);
+    idx = parsec_mca_param_reg_int_name("bind", "threads", "Bind main and worker threads", false, false,
                                   parsec_runtime_bind_threads, &parsec_runtime_bind_threads);
-    parsec_mca_param_reg_int_name("bind", "comm", "Bind the communication thread to physical core <int>."
+    parsec_mca_param_reg_syn_name(idx, "parsec", "bind", true);
+    idx = parsec_mca_param_reg_int_name("bind", "comm", "Bind the communication thread to physical core <int>."
                                                    "-1: do not bind."
                                                    "Warning: binding relies on HWLOC, be careful when using with cgroups",
                                      false, false, comm_binding_parameter, &comm_binding_parameter);
+    parsec_mca_param_reg_syn_name(idx, "parsec", "bind_comm", true);
```

Comment:
I updated the description to explain: the synonyms are reintroduced for the only purpose of complaining if they are used, and they are expected to be inconsistent (reason they got removed in the first place)

Reply:
Since I get you, the synonym registration  with `deprecated=true` does not cause a warning to be issued when the mca is used, but only when calling mca_show_params does a warning get issued. Am I using this wrong?